packages/caliper-samples/network/scalardl/scripts/up.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
It seems like we can pull the repo and `docker-compose up` instead of storing the duplicate files in this repo.